src/lib/hooks/sessionHooks.ts

```diff
@@ -6,7 +6,10 @@ export async function sessionHooks({event}: {event: EventHandler}) {
 			`kinde_${itemKey}`,
 			typeof itemValue === 'string' ? itemValue : (JSON.stringify(itemValue) as string),
 			{
-				path: '/'
+				path: '/',
+				secure: true,
```

Comment:
This won't work on Safari - can we do something like the NextJS SDK https://github.com/kinde-oss/kinde-auth-nextjs/blob/059547d6b6eb88b808aab04c653a380ecdb30ab3/src/session/sessionManager.js#L12

Reply:
Yup